plugin/dns64/README.md

```diff
@@ -27,11 +27,13 @@ Or use this slightly longer form with more options:
 dns64 [PREFIX] {
     [translate_all]
     prefix PREFIX
+    [allow_ipv4]
 }
 ~~~
 
 * `prefix` specifies any local IPv6 prefix to use, instead of the well known prefix (64:ff9b::/96)
 * `translate_all` translates all queries, including responses that have AAAA results.
+* `allow_ipv4` allow DNS queries, ignoring whether they came from over IPv4 or IPv6 network
```

Comment:
Similar here, let's simplify the wording.

```suggestion
* `allow_ipv4` Allow translating queries if they come in over IPv4, default is IPv6 only translation.
```

Reply:
yeah, that sounds much better 👍 fixed as you suggested